### ISSUE TITLE
Setup ServerURL

### ISSUE THREAD

**Issue by 0xErnie** [szelok/twister-android#4]
Hello,

I found twister at the f-droid store. How do I setup I connection to the network? Do I have to start a twisterd somewere in my local network and then add its URL?

Sorry, but I could not find much information on how this works.

Greetings from Hamburg!

**Comment by szelok** [comment]
yes. there should be a twisterd running somewhere - either locally on the android device or on a MAC/PC. in my case, i run only one twisterd on MAC and have all my desktops/tablets/phones to point to that.

Follow this link http://twister.net.co/?page_id=23 to setup your twisterd
